mktxp/cli/config/config.py

```diff
@@ -166,7 +168,7 @@ class MKTXPConfigKeys:
 
     # Feature keys enabled by default
     BOOLEAN_KEYS_YES = {PLAINTEXT_LOGIN_KEY, FE_DHCP_KEY, FE_PACKAGE_KEY, FE_DHCP_LEASE_KEY, FE_IP_CONNECTIONS_KEY, FE_INTERFACE_KEY, 
-                        FE_ROUTE_KEY, FE_DHCP_POOL_KEY, FE_FIREWALL_KEY, FE_NEIGHBOR_KEY, FE_MONITOR_KEY, MKTXP_USE_COMMENTS_OVER_NAMES,
+                        FE_ROUTE_KEY, FE_DHCP_POOL_KEY, FE_FIREWALL_KEY, FE_NEIGHBOR_KEY, FE_DNS_KEY, FE_MONITOR_KEY, MKTXP_USE_COMMENTS_OVER_NAMES,
```

Comment:
this should match the config setting, otherwise would cause a mismatch for existing installations

Reply:
oops. done)